cmd/livegrep-fetch-reindex/main.go

```diff
@@ -316,14 +316,21 @@ func checkoutOne(r *config.RepoSpec) error {
 
 	// git ls-remote --symref origin HEAD
 	g.Go(func() error {
-		remoteOut, remoteErr = callGetGetOutput("git", lsRemoteArgs, username, password)
+		remoteOut, remoteErr = callGetGitOutput("git", lsRemoteArgs, username, password)
 		return remoteErr
 	})
 
 	if err := g.Wait(); err != nil {
 		return err
 	}
 
+	// If there is no remote out, then we have an empty repository.
+	// We should skip and do nothing at this point
+	if remoteOut == nil || len(remoteOut) == 0 {
```

Comment:
`len([]byte(nil))` is zero, so this can just be
```suggestion
	if len(remoteOut) == 0 {
```